docs/rtd_requirements.txt

```diff
@@ -1,23 +1,3 @@
 # This file is used on ReadTheDocs to build the
 # documentation and is separate of tox.ini and setup.cfg
-numpy<1.16
-astropy
-scipy
-pandas
-matplotlib
-sqlalchemy
-scikit-image
-glymur
-drms
-zeep
-pytest
-asdf
-beautifulsoup4
-requests
-python-dateutil
-ruamel.yaml
-sphinx
-sunpy-sphinx-theme
-sphinx-gallery
-sphinx-astropy
 git+https://github.com/hawkowl/towncrier
```

Comment:
we can go back to release towncrier now as a new version was pushed a couple of weeks ago